Framework/DataHandling/src/ProcessBankData.cpp

```diff
@@ -1,6 +1,6 @@
 #include "MantidDataHandling/ProcessBankData.h"
-#include "MantidDataHandling/LoadEventNexus.h"
-#include "MantidDataObjects/Events.h"
+
+//#include "MantidDataObjects/Events.h"
```

Comment:
that's a leftover -- Initially I was not sure It would compile without and then forget to remove